test/blk_device_test.rb

```diff
@@ -24,6 +24,8 @@
 require "y2storage"
 
 describe Y2Storage::BlkDevice do
+  using Y2Storage::Refinements::SizeCasts
```

Comment:
Just convenience. To be able to write

```ruby
vg1.create_lvm_lv("newlv", 1.GiB)
```
down there (lines 135, 169, 214...), instead of
```ruby
vg1.create_lvm_lv("newlv", Y2Storage::DiskSize.GiB(1))
```